Library/Formula/freexl.rb

```diff
@@ -27,4 +27,8 @@ def install
       doc.install "html"
     end
   end
+
+  test do
+    assert File.exist? "#{lib}/libfreexl.a"
```

Comment:
I think we should avoid check file existing in the test. They are basically meaningless.